### PR TITLE
Kubernetes SD: Add node instance label details

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

**Comment by jimmidyson** [comment]
Doc update for prometheus/prometheus#1162

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -323,6 +323,9 @@ Kubernetes API
 * `__meta_kubernetes_service_annotation_<annotationname>`: each service
 annotation from the Kubernetes API
 
+In addition, the `instance` label for node metrics will be set to the node name
```
Comment (juliusv) [inline content/docs/operating/configuration.md]: Also remove the `__meta_kubernetes_node` label documentation above, since that no longer exists?

Comment (jimmidyson) [inline content/docs/operating/configuration.md]: Good spot. Done.

**Comment by juliusv** [comment]
:+1: Thanks!

**Pull request merged**